dydx/dydxPresenters/dydxPresenters/_v4/MarketInfo/dydxMarketInfoViewBuilder.swift

```diff
@@ -31,6 +31,11 @@ private class dydxMarketInfoViewController: HostingViewController<PlatformView,
             let selectedMarketId = request?.params?["market"] as? String ?? dydxSelectedMarketsStore.shared.lastSelectedMarket
             dydxSelectedMarketsStore.shared.lastSelectedMarket = selectedMarketId
             presenter.marketId = selectedMarketId
+            if let sectionRaw = request?.params?["currentSection"] as? String {
+                let section = PortfolioSection(rawValue: sectionRaw) ?? .positions
+                let preselectedSection = Section.allSections.map(\.key).firstIndex(of: section) ?? 0
+                presenter.viewModel?.sections.onSelectionChanged?(preselectedSection)
+            }
```

Comment:
this logic enables the positions/orders/trades tabs to be pre-selected on the market info screen